octoprint_homeassistant/__init__.py

```diff
@@ -648,6 +657,29 @@ def _generate_device_controls(self, subscribe=False):
             _node_id, _node_name, _device_manufacturer, _device_model
         )
 
+        # Connect printer
+        if subscribe:
+            self.mqtt_subscribe(
+                self._generate_topic("controlTopic", "connect", full=True),
+                self._on_connect_printer,
+            )
+
+        self._generate_sensor(
+            topic=_discovery_topic + "/switch/" + _node_id + "_CONNECT/config",
+            values={
+                "name": _node_name + " Connect to printer",
+                "uniq_id": _node_id + "_CONNECT",
+                "cmd_t": "~" + self._generate_topic("controlTopic", "connect"),
+                "stat_t": self._generate_topic("hassTopic", "Connected", full=True),
+                "pl_off": "off",
+                "pl_on": "on",
+                "stat_on": "Connected",
+                "stat_off": "Disconnected",
+                "device": _config_device,
+                "ic": "mdi:power-plug",
```

Comment:
So mostly this look fine, but I think "mdi:power-plug" wouldn't be good here. This has a strong association with another type of device in home automation.

`mdi:lan-connect` or `mdi:check-network` seems like the might be ok choices. I am open to other options you find as well.